src/language.rs

```diff
@@ -4,10 +4,22 @@ use {
     regex::Regex,
     std::collections::HashMap,
     strum::{EnumIter, EnumString},
+    std::env,
 };
 
+#[derive(Clone)]
+pub struct Colors {
+    basic_colors: Vec<Color>,
+    true_colors: Option<Vec<Color>>,
+}
+
+macro_rules! define_colors {
+  ( vec![ $( $color:expr ),+ ] ) => { Colors { basic_colors: vec![$( $color ),+ ], true_colors: None } };
```

Comment:
```suggestion
  ( vec![ $( $color:expr ),+ $(,)? ] ) => { Colors { basic_colors: vec![$( $color ),+ ], true_colors: None } };
```
This allows an optional trailing comma, so that the patterns `vec![foo, bar, ]` and `vec![foo, bar]` are both accepted.

Reply:
reworked define_colors to not utilize vec.